frontend/src/main.ts

```diff
@@ -21,7 +21,7 @@ dom.watch();
 const app = createApp(App);
 
 app.directive('hljs', {
-  mounted: (ele: any, binding: DirectiveBinding) => {
+  mounted: (ele, binding: DirectiveBinding) => {
```

Comment:
```suggestion
  mounted: (ele: HTMLElement, binding: DirectiveBinding) => {
```
Is there any reason we do not define the type here?

Reply:
Thanks for catching this! I think it would be good to add the type declaration here as well.